content/rc/rc-quickstart.md

```diff
@@ -14,74 +14,39 @@ If you're new to Redis Enterprise Cloud, this quick start helps you get up and r
 
 You'll learn how to:
 
-1.  Create a free subscription
+1.  Create an account with a free subscription and database
```

Comment:
```suggestion
1.  Create an account, a free subscription, and a database
```

Reply:
Even though these are are all done together in one process, it might be good to word it so people understand that they can be done separately.